### PR TITLE
Minor cleanup

### PR DESCRIPTION
**Pull request by WilliamJamieson**
Clean-up flake8 and docs configs

### CODE REVIEW

**Comment by perrygreenfield** [comment]
Might be good to briefly summarize using tomli vs toml in the previous version for someone like me that isn't familiar with the need to change it.

**Comment by WilliamJamieson** [comment]
> Might be good to briefly summarize using tomli vs toml in the previous version for someone like me that isn't familiar with the need to change it.

[tomli](https://github.com/hukkin/tomli) was the library that was chosen in [PEP 680](https://peps.python.org/pep-0680/) to be incorporated into Python 3.11+ as the standard toml parsing library over the "toml" package. Thus in Python 3.11+ toml parsing will become a standard feature of Python whose interface and behavior will match the tomli package. So it makes sense to switch over to the library which will become the standard, as it involved such a minor change.

**Comment by WilliamJamieson** [comment]
Note that the "toml" package/library was originally created and maintained by PyPi to handle pyproject.toml files for pip. However, PyPI has since switched pip over to using tomli exclusively, and so support for the "toml" package has mostly ended, for example it does not full support TOML v1.0.0 (of the format) unlike tomli which does.

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -11,6 +11,17 @@ repos:
   - id: end-of-file-fixer
   - id: trailing-whitespace
 
+- repo: https://github.com/asottile/pyupgrade
+  rev: v2.37.3
+  hooks:
+    - id: pyupgrade
+      args: ["--py38-plus"]
+
+- repo: https://github.com/fsouza/autoflake8
```
Comment (eslavich) [inline .pre-commit-config.yaml:20]: The autoflake8 repo has been archived with instructions in the readme to use autoflake instead:

image

**Review by eslavich — COMMENTED** [review]
Just the thing about autoflake, looks good otherwise

**Comment by WilliamJamieson** [comment]
> Just the thing about autoflake, looks good otherwise

Wow, that happened (recently) after I started using it for other ASDF projects. I guess I am behind and need to update all those ones now.

**Review by zacharyburnett — APPROVED** [review]
LGTM

**Pull request merged**